templates/download/desktop/thank-you.html

```diff
@@ -40,240 +41,328 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Desktop</h1>
-            <input class="p-slider" min="0" max="125" value="3" step="1" id="amount-things" type="range">
-            <label for="amount-things-input" class="u-off-screen">Contribution to IOT projects in US dollars</label>
-            <input class="p-slider__input" style="min-width: 4rem;" min="0" max="125" id="amount-things-input" name="amount_3" value="3" tabindex="3" type="number">
-            <input type="hidden" name="item_name_3" value="Ubuntu for things">
-          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-3 col-medium-6">
+      <div class="row">
+        <div class="col-6 col-medium-2">
+          <h3>
+            <a href="https://canonical.com/lxd">
+              {{
+                image (
+                  url="https://assets.ubuntu.com/v1/84ed7eb1-LXD-logo.svg",
+                  alt="LXD",
+                  width="80",
+                  height="42",
+                  hi_def=True,
+                  loading="auto"
+                ) | safe
+              }}
+            </a>
+          </h3>
+        </div>
+        <div class="col-6 col-medium-4">
+          <p>Fast, dense, and secure container and VM management at any scale.</p>
+          <hr class="p-rule" />
           <p>
-            I want a secure, upgradeable Internet of Things, powered by Ubuntu.
+            <a href="https://canonical.com/lxd">Learn about LXD&nbsp;&rsaquo;</a>
           </p>
-        </section>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
 
-        <section id="ubuntu-for-cloud" class="u-sv3">
-          <p class="p-heading--4">Ubuntu Server & Cloud</p>
-          <div class="p-slider__wrapper">
-            <label for="amount-cloud" class="u-off-screen">Contribution to server and cloud projects in US dollars</label>
-            <input class="p-slider" min="0" max="125" value="3" step="1" id="amount-cloud" type="range">
-            <label for="amount-cloud-input" class="u-off-screen">Contribution to server and cloud projects in US dollars</label>
-            <input class="p-slider__input" style="min-width: 4rem;" min="0" max="125" id="amount-cloud-input" name="amount_2" value="3" tabindex="2" type="number">
-            <input type="hidden" name="item_name_2" value="Ubuntu for cloud computing">
-          </div>
-          <p>
-            I want Ubuntu Server running my servers, cloud and as a guest everywhere.
-          </p>
-        </section>
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Help us to keep Ubuntu free to download, share and use by contributing to...</h2>
+    </div>
+    <div class="col">
+      <noscript>
+        <p>Contributions require JavaScript. Please enable JavaScript if you want to contribute.</p>
+      </noscript>
+      <form class="contribute__options" action="https://www.paypal.com/cgi-bin/webscr" id="contributions-form" method="post" target="_top">
```

Comment:
The slider colors look a bit different from the design depending on the browser, not sure if it's a known issue

on firefox:
![image](https://github.com/canonical/ubuntu.com/assets/17607612/4b1f0422-87ba-43d6-8fdc-764f71062934)

on chrome:
![image](https://github.com/canonical/ubuntu.com/assets/17607612/f7785b7a-9297-4b59-b8f5-bfabaf6614e5)

Reply:
Good point, I opened a [ticket in Vanilla](https://github.com/canonical/vanilla-framework/issues/5025). I couldn't do a proper fix, it seems that when I fix it in one browser the other fails even when I target the browser with Sass. Not sure what's the best solution.

Reply:
Ok, I think I fixed it. Temporarily